### DISCUSSION TITLE
Welcome to python-polylith Discussions!

### DISCUSSION THREAD

**Discussion by DavidVujic** [DavidVujic/python-polylith#81] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gadieichhorn** [comment]
hello all, just starting my first project and already stuck.
I followed the setup steps to generate a project with a workspace.
when I run the tests I get this error
> poetry run pytest

```ImportError while importing test module '/Users/gadi.eichhorn@pexapark.com/Projects/pexapark/quant/polylith/components/gateway/test/quant/gateway/test_core.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/homebrew/Cellar/python@3.12/3.12.0/Frameworks/Python.framework/Versions/3.12/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
components/gateway/test/quant/gateway/test_core.py:1: in <module>
    from quant.gateway import core
E   ModuleNotFoundError: No module named 'quant'
```

**Reply by gadieichhorn** [reply]
not sure this is related but I named the repo polylith and the workspace quant

```[tool.poetry]
name = "polylith"
version = "0.1.0"
description = ""
```

```
[tool.polylith]
namespace = "quant"
git_tag_pattern = "stable-*"
```

**Reply by DavidVujic** [reply]
The reason was because of missing usages in the `packages` section of `[tool.poetry]` in the `pyproject.toml` file at the root level.

@gadieichhorn  and I solved this during a Video chat earlier today. 😄
